### ISSUE TITLE
Unable to start on Gnome/Wayland since version 1.11.33

### ISSUE THREAD

**Issue by waruiji** [flathub/im.riot.Riot#390]
Here is the relevant log information

```
[72:0806/140606.072420:ERROR:gl_display.cc(520)] EGL Driver message (Critical) : eglCreateImage failed with 0x00003009
[72:0806/140606.072470:ERROR:scoped_egl_image.cc(23)] Failed to create EGLImage: EGL_BAD_ALLOC
[72:0806/140606.072485:ERROR:native_pixmap_egl_binding.cc(152)] Unable to initialize binding from pixmap
[72:0806/140606.072503:ERROR:ozone_image_backing.cc(189)] OzoneImageBacking::ProduceSkiaGanesh failed to create GL representation
[72:0806/140606.072518:ERROR:shared_image_manager.cc(224)] SharedImageManager::ProduceSkia: Trying to produce a Skia representation from an incompatible mailbox.
[72:0806/140606.072562:ERROR:gpu_service_impl.cc(1010)] Exiting GPU process because some drivers can't recover from errors. GPU process will restart shortly.
[3:0806/140606.075394:ERROR:gpu_process_host.cc(954)] GPU process exited unexpectedly: exit_code=8704
[159:0806/140606.112575:ERROR:gl_display.cc(520)] EGL Driver message (Critical) : eglCreateImage failed with 0x00003009
[159:0806/140606.112620:ERROR:scoped_egl_image.cc(23)] Failed to create EGLImage: EGL_BAD_ALLOC
[159:0806/140606.112637:ERROR:native_pixmap_egl_binding.cc(152)] Unable to initialize binding from pixmap
[159:0806/140606.112665:ERROR:ozone_image_backing.cc(189)] OzoneImageBacking::ProduceSkiaGanesh failed to create GL representation
[159:0806/140606.112682:ERROR:shared_image_manager.cc(224)] SharedImageManager::ProduceSkia: Trying to produce a Skia representation from an incompatible mailbox.
[159:0806/140606.112741:ERROR:gpu_service_impl.cc(1010)] Exiting GPU process because some drivers can't recover from errors. GPU process will restart shortly.
[120:0806/140606.114399:ERROR:command_buffer_proxy_impl.cc(128)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.
[3:0806/140606.115362:ERROR:gpu_process_host.cc(954)] GPU process exited unexpectedly: exit_code=8704
[224:0806/140606.151609:ERROR:gl_display.cc(520)] EGL Driver message (Critical) : eglCreateImage failed with 0x00003009
[224:0806/140606.151647:ERROR:scoped_egl_image.cc(23)] Failed to create EGLImage: EGL_BAD_ALLOC
[224:0806/140606.151668:ERROR:native_pixmap_egl_binding.cc(152)] Unable to initialize binding from pixmap
[224:0806/140606.151682:ERROR:ozone_image_backing.cc(189)] OzoneImageBacking::ProduceSkiaGanesh failed to create GL representation
[224:0806/140606.151701:ERROR:shared_image_manager.cc(224)] SharedImageManager::ProduceSkia: Trying to produce a Skia representation from an incompatible mailbox.
[224:0806/140606.151739:ERROR:gpu_service_impl.cc(1010)] Exiting GPU process because some drivers can't recover from errors. GPU process will restart shortly.
[120:0806/140606.153249:ERROR:command_buffer_proxy_impl.cc(128)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.
[3:0806/140606.154125:ERROR:gpu_process_host.cc(954)] GPU process exited unexpectedly: exit_code=8704
[120:0806/140606.176466:ERROR:command_buffer_proxy_impl.cc(128)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.
```

Gnome version is 44.3 on Arch Linux with latest updates applied (as of 2023-08-06).
GPU is AMD Radeon RX 6700 XT using `amdgpu` driver.

Switching to `XWayland` by disabling `Wayland` support works on v1.11.37.
Switching back to v1.11.33 (commit `6b24295a5fa7e4e4c34177adf7c02d62947d46c1d6d048ef54b50367a1ba3ba7`) works with `Wayland`.

**Comment by byquanton** [comment]
Can you check if the error persists if you delete the ~/.var/app/im.riot.Riot/config/Element/GPUCache Folder?

**Comment by waruiji** [comment]
Unfortunately that doesn't help. I also tried with a completely new profile using cli option.
The tray icon is fine, I receive notifications, but the main window is not rendered.

**Comment by felbinger** [comment]
Same problem here, might be related to nvidia drivers, when using `nouveau` driver it still worked...

**Comment by besendorf** [comment]
For me it also does not start on Arch Linux. Where can I access those logs? stdout does not show anything.
I use AMD drivers.

**Comment by waruiji** [comment]
A temporary workaround I found is to disable GPU rendering using `--disable-gpu` command line option, e.g.

`flatpak run im.Riot.riot --disable-gpu`

**Comment by waruiji** [comment]
Seems to have been fixed in version 1.11.47.
